app/src/main/java/com/github/se/studybuddies/ui/groups/CreateGroup.kt

```diff
@@ -0,0 +1,67 @@
+package com.github.se.studybuddies.ui.groups
+
+import android.net.Uri
+import androidx.activity.compose.rememberLauncherForActivityResult
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.github.se.studybuddies.navigation.NavigationActions
+import com.github.se.studybuddies.navigation.Route
+import com.github.se.studybuddies.ui.SecondaryTopBar
+import com.github.se.studybuddies.ui.settings.SetProfilePicture
+import com.github.se.studybuddies.viewModels.GroupViewModel
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+
+@Composable
+fun CreateGroup(groupViewModel: GroupViewModel, navigationActions: NavigationActions) {
+  val nameState = remember { mutableStateOf("") }
+  val photoState = remember { mutableStateOf(Uri.EMPTY) }
+
+  LaunchedEffect(key1 = true) {
```

Comment:
Same as in `CreateAccount`, should `key1` really be set to `true`?

Reply:
I must admit I kind of wrote this blindly (code from ChatGPT) because I didn't really understand it. This may be why I had some issues with the pages not loading.